resources/kcp/charts/component-reconcilers/templates/authorization.yaml

```diff
@@ -0,0 +1,22 @@
+{{ $global := . }}
+{{- range $component := .Values.global.components }}
+{{ $subchart := printf "%s-component-reconciler" $component }}
+{{ if not (hasKey $global.Subcharts $subchart) }}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
```

Comment:
I think you also need RequestAuthentication object defined for the same workload?

Reply:
Yes, I missed it. Thanks!

Reply:
Should I use the `RequestAuthentication` if I'm not using JWT? Principals are defined a few lines under ([here](https://github.com/kyma-project/control-plane/pull/1024/files#diff-af952c9ab6230be60c72a211a4ad7ea06f0d38fcb92854df1dc19f45fe9a2cc9R19))

Reply:
👍